docs/source/en/api/pipelines/text_to_video_zero.md

```diff
@@ -257,4 +270,4 @@ You can filter out some available DreamBooth-trained models with [this link](htt
 	- __call__
 
 ## TextToVideoPipelineOutput
-[[autodoc]] pipelines.text_to_video_synthesis.pipeline_text_to_video_zero.TextToVideoPipelineOutput
\ No newline at end of file
+[[autodoc]] pipelines.text_to_video_synthesis.pipeline_text_to_video_zero.TextToVideoPipelineOutput
```

Comment:
We need to add ` TextToVideoZeroSDXLPipeline` in the docs too otherwise they won't reflect :-).